sdk/servicebus/service-bus/test/utils/envVarUtils.ts

```diff
@@ -36,28 +34,38 @@ let envVars: any;
 /**
  * Utility to return cached map of environment variables,
  * or create and return one from configured values if not existing.
+ *
+ * The utility helps use the right environment variable key based on targetted platform and type.
+ * Specifically, we use different Service Bus namespaces for browser Vs node test runs.
+ * Thus, the connection string value is retrieved from `SERVICE_BUS_CONNECTION_STRING_BROWSER`
+ * environment variable key for browser, and from `SERVICE_BUS_CONNECTION_STRING` for Node.
  */
 export function getEnvVars(): { [key in EnvVarKeys]: any } {
   if (envVars != undefined) {
     return envVars;
   }
 
+  let serviceBusConnectionStringEnvVarKey: string = EnvVarKeys.SERVICEBUS_CONNECTION_STRING.valueOf();
+
+  if (!isNode) {
+    serviceBusConnectionStringEnvVarKey =
+      EnvVarKeys.SERVICEBUS_CONNECTION_STRING.valueOf() + "_BROWSER";
```

Comment:
Nit:

This can be `serviceBusConnectionStringEnvVarKey += "_BROWSER"`

Reply:
Updated